### ISSUE TITLE
Keep/reshow function intellisense on subsequent arguments

### ISSUE THREAD

**Issue by turb0mint** [DaxStudio/DaxStudio#366]
Currently intellisense for functions is only show immediately after the first parentheses when writing a function. 

I'm requesting that this be updated so that the intellisense for the function in scope reshows the function definition when moving to the next argument in the function similar to how excel does this after each argument separating comma.

This would also help a user identify/debug long dax expressions with multiple nested functions.

![image](https://user-images.githubusercontent.com/35158378/85307358-ba846b80-b4a7-11ea-80a8-664b05b915da.png)

**Comment by dgosbell** [comment]
Thanks for logging this. It is something that we are aware of and have been looking into trying to fix. However it's not an easy fix as doing this properly so that it works in all circumstances really requires us to build a full DAX parser and semantic analyzer which is a lot of work. We've started looking at this, but it's going to take a while.

**Comment by turb0mint** [comment]
Yeah, I can see how this could be a complex addition and I didnt notice that there was a TODO in the class to add a ctrl-space function.

In the interim, could we have it that when the mouse is hovered over function that the definiton shows up, I constantly find myself deleting and retyping the left parentheses to see the definition.
